README.md

```diff
@@ -28,54 +28,47 @@ Require the library
 require 'graphql/batch'
 ```
 
-Define a GraphQL::Batch::Query derived class. Use group_key to specify which queries can be reduced into a batch query, and an execute class method which makes that batch query and passes the result to each individual query.
+Define a custom loader, which is initialized with arguments that are used for grouping and an execute method for doing the batch query. The load method can be overriden to coerce the type of the key.
 
 ```ruby
-class FindQuery < GraphQL::Batch::Query
-  attr_reader :model, :id
+class RecordLoader < GraphQL::Batch::Loader
```

Comment:
I thought calling it a loader was appropriate, since this library is similar to the [dataloader js library](https://github.com/facebook/dataloader).  I don't see why Loader isn't an appropriate name, since it is loading a record.